### PR TITLE
Delete requests pricing faq

### PR DESCRIPTION
**Pull request by adilansari**

### CODE REVIEW

**Review by garrensmith — APPROVED** [review]

**Review by himank — DISMISSED** [review]

File: docs/pricing/index.md [hunk 1]
```diff
@@ -102,6 +102,13 @@ GetObjectTagging, HeadBucket, HeadObject
 
 </details>
 
+<details>
```
Comment (ovaistariq) [inline docs/pricing/index.md:105]: I think we should just expand the Class A and B FAQ and add a line about DELETE and CANCEL requests.

Comment (adilansari) [reply docs/pricing/index.md:105]: Ok, I think deletion is a valid Q that can have a verbose answer.

image

Comment (adilansari) [reply docs/pricing/index.md]: Merged to existing FAQ

Comment (ovaistariq) [reply docs/pricing/index.md]: I didn't want to highlight that as a thing right now.

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**